api_specs/v2/component_analyses.yaml

```diff
@@ -120,74 +272,84 @@ components:
             properties:
               vendor_cve_ids:
                 type: string
+                example: "CVE-1234"
```

Comment:
Vendor cve id is not actual cve id. It starts with SNYK-..

Reply:
Ack!

Reply:
I guess you misinterpreted my comment. A vulnerability should have both id(vendor_id) as well as list of cve_ids associated with it. Please check https://github.com/fabric8-analytics/f8a-server-backbone/blob/master/src/v2/openapi.yaml#L145

Single vendor vulnerability can have multiple CVES.

Reply:
No field changes are carried out in Existing API, 
for Batch all, field name changed from `vendor_cve_ids -> id` . This will have SYNK id